diff --git a/3rdparty/CMakeLists.txt b/3rdparty/CMakeLists.txt
index ac213bb7d3..f1fc94733e 100644
--- a/3rdparty/CMakeLists.txt
+++ b/3rdparty/CMakeLists.txt
@@ -267,7 +267,7 @@ else()
 		if (WIN32)
 			set(FFMPEG_LIB_DIR "ffmpeg/windows/x86_64")
 			target_link_libraries(3rdparty_ffmpeg INTERFACE "Bcrypt.lib")
-		elseif(CMAKE_SYSTEM MATCHES "Linux")
+		elseif(TRUE)
 			set(FFMPEG_LIB_DIR "ffmpeg/linux/x86_64")
 		elseif(APPLE)
 			set(FFMPEG_LIB_DIR "ffmpeg/macos/x86_64")
